### ISSUE TITLE
set up CI workflows for validating submissions

### ISSUE THREAD

**Issue by nickreich** [reichlab/variant-nowcast-hub#61]
We should be able to set up all of the standard file-format validations and then also add the custom validations from here: https://github.com/reichlab/variant-nowcast-hub/issues/55

**Comment by nickreich** [comment]
The main validations are in place, now just waiting on the custom ones from #55

**Comment by nickreich** [comment]
Going to close this, as we are tracking this in another issue now. #75
